taca/utils/filesystem.py

```diff
@@ -4,6 +4,7 @@
 import shutil
 
 RUN_RE = '^\d{6,8}_[a-zA-Z\d\-]+_\d{2,}_[AB0][A-Z\d\-]+$'
+RUN_RE_ONT = '^(\d{8})_(\d{4})_([0-9a-zA-Z]+)_([0-9a-zA-Z]+)_([0-9a-zA-Z]+)$' 
```

Comment:
The `minion` pattern is missing here. The script will not handle them due to line 62 and 79.

Reply:
The regex should be compatible with both minion and promethion runs. At least when I check it on the preproc I couldn't see any problem. Is there a specific case I am not testing?

Reply:
OK thanks! I think I didn't see the difference between this one and line 145.